### PR TITLE
remove interactionName parameter from EndInteraction method.

### PR DESCRIPTION
**Pull request by cmarcelk**
- also changed version from 2.0.0 to 2.0.0-pre, since we have more changes coming before the 2.0.0 official release to a repo.

### CODE REVIEW

**Review by jkashimba — COMMENTED** [review]
We should support both forms of end interaction.  When name is not specified all interactions for the target(s) are ended, when specified by name only the named interaction is ended for the target(s).

**Comment by cmarcelk** [comment]
@jkashimba With my understanding that the target is an interaction urn, the name parameter duplicates the interaction urn, and without the name parm only that one interaction is ended. I thought the target can be only an interaction urn and not a device urn, per https://bitbucket.org/relaypro/relay_erlang/src/master/priv/wf_api_end_interaction_request.json, or do I misunderstand? Or are you talking about ending only one interaction in the target array?

**Comment by jkashimba** [comment]
@cmarcelk let me take a closer look.  i was just comparing to our internal handling, and my understanding may be out of date as well.  i'll touch base with adam and circle back on this.

File: RelayDotNet/Relay.cs [hunk 1]
```diff
@@ -860,22 +860,18 @@ public async void StartInteraction(IRelayWorkflow relayWorkflow, string sourceUr
         /// </summary>
         /// <param name="relayWorkflow">the workflow.</param>
         /// <param name="sourceUri">the device that you would like to end an interaction with.</param>
```
Comment (cmarcelk) [inline RelayDotNet/Relay.cs]: This should say "interaction" instead of "device".

Comment (cmarcelk) [reply RelayDotNet/Relay.cs]: Fixed.

**Pull request merged**